packages/cli/src/CommandsLogic.ts

```diff
@@ -505,20 +521,23 @@ export class CommandsLogic {
     }
     await registerForwarderForGsn(fInstance, options)
 
-    let ttInstance: Contract | undefined
-    if (deployOptions.deployTestToken ?? false) {
-      ttInstance = await this.getContractInstance(TestToken, {}, undefined, { ...options }, deployOptions.skipConfirmation)
-      console.log('Setting minimum stake of 1 TestToken on Hub')
-      await rInstance.methods.setMinimumStakes([ttInstance.options.address], [1e18.toString()]).send({ ...options })
+    let stakingTokenAddress = deployOptions.stakeManagerAddress
+    if (stakingTokenAddress == null) {
```

Comment:
I don't think it is wise to deploy a contract by default. I think the flag should stay. This differentiates running this command for tests or to actually deploy the contracts. We still didn't migrate to hardhat so let's maintain this command reasonable.